source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=====================================
+Set maxTimeMS for Aggregation Queries
+=====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the ``maxTimeMS`` parameter on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|.
+
+About This Task 
+---------------
+
+If you already specified a ``maxTimeMS`` value in the :ref:`Compass Settings 
+panel <compass-interface-settings>`, the :ref:`command line <cli-options>`, or 
+a :ref:`configuration file <config-file-options>`, the value you enter for 
+:guilabel:`Max Time MS` value must be lower than the limit you previously 
+specified.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`More Options`.
+      
+      On the :guilabel:`Aggregations` tab, click :guilabel:`More Options`
+
+      .. figure:: /images/compass/agg-builder-click-more-options.png
+         :alt: More Options dropdown.
+
+   .. step:: Specify a Max Time MS value.
+
+      Next to the :guilabel:`Max Time MS` field, enter a numeric value to set as 
```

Comment:
Gotcha. I'm leaning towards changing to 

> :guilabel:`Max Time MS`

but one exception is that in the second paragraph under "About This Task", I'm referring to it by the camel case parameter name since it's talking about the corresponding option on the interface settings. Do you think we should keep it as is there?

Reply:
Right, I see the concern here. Hmm... In that case, I think it's ok to keep it camel case in those instances.